### PR TITLE
changed progress bar and utilization color for easier indication

### PR DESCRIPTION
**Pull request by thiru85**
*Issue #, if available*: https://github.com/awslabs/eks-node-viewer/issues/141

*Description of changes:*

- Made the change for the progress bar to go from green to red. Also made the text indicator reflect the same colors.
- After review, added flag _-invert-colors_ to let user choose

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: pkg/model/uimodel.go [hunk 1]
```diff
@@ -53,8 +53,8 @@ func NewUIModel(extraLabels []string) *UIModel {
 	pager.ActiveDot = activeDot
 	pager.InactiveDot = inactiveDot
 	return &UIModel{
-		// red to green
-		progress:    progress.New(progress.WithGradient("#ff0000", "#04B575")),
```
Comment (tzneal) [inline pkg/model/uimodel.go:57]: Can you add a flag for swapping the colors (e.g. `--invert-colorbar`?), with the current coloring being the default?  I picked red to green intentionally as low utilization isn't the goal and thought it made more sense to show red in that case.

File: ATTRIBUTION.md [hunk 2]
```diff
@@ -1,5982 +0,0 @@
-# Open Source Software Attribution
```
Comment (tzneal) [inline ATTRIBUTION.md:1]: Can you revert this deletion?

File: pkg/model/uimodel.go (hunk 1 above)
Comment (thiru85) [reply pkg/model/uimodel.go:57]: Yup, will delete this PR and create a new one - added this flag just now

File: ATTRIBUTION.md (hunk 2 above)
Comment (thiru85) [reply ATTRIBUTION.md]: re-ran make generate and it fixed this

File: pkg/model/uimodel.go (hunk 1 above)
Comment (thiru85) [reply pkg/model/uimodel.go:57]: Sorry, shouldn't have done that - apologies. Let me know if this works.

File: hack/ATTRIBUTION.md
Comment (tzneal) [inline hack/ATTRIBUTION.md:1]: I don't think this needs to be here.

File: pkg/model/uimodel.go [hunk 3]
```diff
@@ -69,7 +78,7 @@ func (u *UIModel) Init() tea.Cmd {
 	return nil
 }
 
-var green = lipgloss.NewStyle().Foreground(lipgloss.Color("#04B575")).Render
```
Comment (tzneal) [inline pkg/model/uimodel.go:72]: Why was this changed?

Comment (thiru85) [reply pkg/model/uimodel.go:72]: I made this change to reflect the usage as well, i.e, red for highly utilized. I can add this too, as part of the flag if that works?

File: hack/ATTRIBUTION.md
Comment (thiru85) [reply hack/ATTRIBUTION.md:1]: This is part of the build artifact I think. Will remove in the next commit.

File: pkg/model/uimodel.go (hunk 3 above)
Comment (tzneal) [reply pkg/model/uimodel.go:72]: Yes, I want to keep the default colors as-is and optionally allow someone to swap them.

**Comment by tzneal** [comment]
Closing this for now due to no activity, feel free to re-open when you get some time.
